combine-events/index.js

```diff
@@ -5,19 +5,27 @@ const {
   merge,
   sample,
   withRegion,
+  is,
 } = require('effector');
 const { readConfig } = require('../library');
 
+const throwError = (message) => {
+  throw new Error(message);
+};
+
 function combineEvents(argument) {
-  const { events, loc, name = 'unknown' } = readConfig(argument, [
-    'events',
+  const {
+    events,
+    reset,
+    target: givenTarget,
```

Comment:
```suggestion
    target = createEvent({ name: `${name}Target`, loc }),
```

And, please, move `sid` and `loc` and `name` before all other props.

```
    name = ‘unknown’,
```

Reply:
done